chrome/content/prefs.js

```diff
@@ -14,10 +16,35 @@ if (window && window.arguments) {
 }
 
 var otrPref = {
+  _inited: false,
 
   onload: function() {
+    this._inited = true;
+    let otrPrefs = document.getElementById("otrPrefs");
+    let preference = document.getElementById("extensions.otr.preferences.selectedTabIndex");
+    if (preference.value !== null)
+      otrPrefs.selectedIndex = preference.value;
+
+    otrPref.otrTabInit();
+    if (coniks.Enabled) {
+      otrPref.coniksTabInit();
+    } else {
+      document.getElementById("coniksTab").hidden = true;
```

Comment:
Hidden should be the default, inlined as an attribute on the element.  Then just,

```
if (coniks.isEnabled)
  otrPref.coniksTabInit();
```

and

```
document.getElementById("coniksTab").hidden = false;
```

in `coniksTabInit`